src/UI/components/map/layers/filters/dateFilter.test.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { filterHandler, initialState } from '../../../../state/map/mapSlice';
+import { fireEvent, render } from '../../../../test_config/render';
+import DateFilter from './dateFilter';
+
+jest.mock('@mui/x-date-pickers', () => {
+  return {
+    DatePicker: jest.requireActual('@mui/x-date-pickers').DesktopDatePicker,
+    LocalizationProvider: jest.requireActual('@mui/x-date-pickers')
+      .LocalizationProvider,
+  };
+});
```

Comment:
Why have you done this? It looks like you've mocked the date pickers, only to use the actual ones anyway?

Reply:
It was needed because the `DatePicker` dynamically switches between the desktop version or the mobile version, you have to force it to use the desktop version. The default import of `@mui/x-date-pickers` meant I needed to do a passthrough mock on the LocalizationProvider.

Reply:
Oh flip